templates/server/src/main/resources/archetype-resources/core/src/main/resources/application.properties

```diff
@@ -39,6 +39,10 @@ spring.datasource.username=${rootArtifactId}
 # spring.jpa.database-platform=org.hibernate.dialect.MySQL5Dialect
 # spring.datasource.driver-class-name=org.mariadb.jdbc.Driver
 spring.datasource.username=${rootArtifactId}
+#elseif ($dbType == 'hana')
```

Comment:
Property _spring.jpa.database_ is set to **hana**, which is not a valid value. As of now _hana_ database cannot be set explicitly using this property. So from the list of available values only **DEFAULT** seems to be a sensible choice and using it Hibernate is able to resolve the dialect correctly. So it is better to set this property to **DEFAULT**

Reply:
Setting HANA via `spring.jpa.database` will be possible starting with Spring 5.1 (see Jira item https://jira.spring.io/browse/SPR-16460)

Reply:
Thanks for all your valuable feedback. Also great seeing how hana evolves and becomes first class citizen by all frameworks :+1: 
With this information, I would suggest not to change anything. We just leave it like this and it will get fixed in the future when the update to spring 5.1 takes place.

Reply:
@hohwille but with the current setting application failed to startup. So until Spring 5.1 is available we need to provide a valid value that works for Hana.